### PR TITLE
[Shared dev] [#162254] Open "Fix problem" in modal

### PR DESCRIPTION
**Pull request by LeticiaErrandonea**
# Release Notes
* [Shared dev] [#162254] Open "Fix problem" in modal

# Screenshot
image

# Additional Context
## Pending work
- [ ] Specs

# Accessibility
- [ ] Did you scan for accessibility issues?
- [ ] Did you check our accessibility goal checklist?
- [ ] Did you add accessibility [specs](https://github.com/dequelabs/axe-core-gems/blob/develop/packages/axe-core-rspec/README.md)?

### CODE REVIEW

File: app/views/problem_reservations/edit.html.haml [hunk 1]
```diff
@@ -1,37 +1,85 @@
...
+
+
+- else
+  %h2= @order_detail.product
+
+  %p.alert.alert-danger
+    = text("instructions")
+    = text("accessories") if @order_detail.accessories?
+
+  -# TODO: Move this to partial to avoid code repetition?
```
Comment (LeticiaErrandonea) [inline app/views/problem_reservations/edit.html.haml:57]: Use partial?

File: app/views/problem_reservations/edit.html.haml [hunk 2]
```diff
@@ -1,37 +1,85 @@
...
+
+          .row
+            .span7
+              = f.input :actual_duration_mins, hint: text("views.reservations.reservation_fields.duration_hint"), input_html: { class: "timeinput", autofocus: true }
+
+      %ul.inline
+        %li= f.submit t("shared.save"), class: 'btn'
+
+  .modal-footer
+    -# TODO: i18n
```
Comment (LeticiaErrandonea) [inline app/views/problem_reservations/edit.html.haml:46]: i18n

File: app/controllers/problem_reservations_controller.rb [hunk 3]
```diff
@@ -9,19 +9,26 @@ class ProblemReservationsController < ApplicationController
...
   before_action { @active_tab = "reservations" }
 
+  layout -> { modal? ? false : "application" }
+
   delegate :editable?, :resolved?, to: :problem_reservation_resolver
 
   def edit
-    render :resolved if !editable? && resolved?
+    if !editable? && resolved?
+      render :resolved
```
[the remark this answers is not in the document]
Comment (giladshanan) [reply app/controllers/problem_reservations_controller.rb:18]: if possible, please add a spec

**Comment by giladshanan** [comment]
I added a link to this WIP PR in the redmine ticket in case we pick this up in the future

**Pull request closed without merging**